app/get_proc_arg.go

```diff
@@ -92,13 +92,13 @@ func getProcArg(kubeClient *clientset.Clientset, managerImage, name string) (str
 	case ArgFlexvolumePluginDir:
 		dir, err := discoverFlexvolumeDir(kubeClient, managerImage)
 		if err != nil {
-			return "", errors.Wrap(err, "Failed to get arg volume-plugin-dir")
+			return "", errors.Wrap(err, "Failed to get arg volume-plugin-dir. You can override the detection by setting arg flexvolume-dir in https://github.com/rancher/longhorn/blob/master/deploy/longhorn.yaml#L317")
```

Comment:
Just say `Failed to get arg volume-plugin-dir. Need to specify \"--flexvolume-dir\" in your Longhorn deployment yaml`. Because the line number of yaml file can change after an upgrade.

Same below.